config/routes.rb

```diff
@@ -12,11 +12,20 @@
 
   get 'up' => 'rails/health#show', as: :rails_health_check
 
-  get 'registration', to: 'users#new'
+  # authorization flows
+  get 'registration',  to: 'users#new'
   post 'registration', to: 'users#create'
 
-  get 'login', to: 'sessions#new'
-  post 'login', to: 'sessions#create'
+  get 'login',         to: 'sessions#new'
+  post 'login',        to: 'sessions#create'
 
-  delete 'logout', to: 'sessions#destroy'
+  delete 'logout',     to: 'sessions#destroy'
+
+  # visitable pages
+  resources :users
+  resources :friendships do
+    collection do
```

Comment:
This prevents the composite primary key from being printed in the URL, but means it needs to be submitted in the form body.

Reply:
Interesting, so that means that we'll need both user's id's available for the form no? Or would it better to have the form submit action trigger a search based on the users unique name?

Reply:
No, we only need the friend to be requested ID, because we already will have `current_user` at the endpoint layer. So it will be [Pseudocode] `Friendship.create(user: current_user, friend: friend_param)`